root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,196 @@
+import * as manifest from '../static/manifest';
+import releaseGroupType from '../utility/releaseGroupType';
+
+import ReleaseGroupLayout from './ReleaseGroupLayout';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +eligibleForCleanup: boolean,
+  +mostPopularReview: CritiqueBrainzReviewT,
+  +mostRecentReview: CritiqueBrainzReviewT,
+  +numberOfRevisions: number,
+  +pager: PagerT,
+  +releaseGroup: ReleaseGroupT,
+  +releases: $ReadOnlyArray<$ReadOnlyArray<ReleaseT>>,
+  +wikipediaExtract: WikipediaExtractT,
+|};
+
+function buildReleaseStatusTable($c, releaseStatusGroup) {
+  const status = releaseStatusGroup[0].status;
+  return (
+    <>
+      <tr className="subh">
+        {$c.user_exists ? <th /> : null}
+        <th colSpan={$c.session && $c.session.tport ? 9 : 8}>
+          {status && status.name
+            ? lp_attributes(status.name, 'release_status')
+            : l('(unknown)')}
+        </th>
+      </tr>
+      {releaseStatusGroup.map((release, index) => (
+        <tr className={loopParity(index)} key={release.id}>
+          {$c.user_exists
+            ? (
+              <td>
+                <input
+                  name="add-to-merge"
+                  type="checkbox"
+                  value={release.id}
+                />
+              </td>
+            ) : null}
+          <td>
+            <EntityLink entity={release} />
+          </td>
+          <td>{release.combined_format_name || l('[missing media]')}</td>
+          <td>{release.combined_track_count || '-'}</td>
+          <td>
+            <ReleaseDates events={release.events} />
+          </td>
+          <td>
+            <ReleaseCountries events={release.events} />
+          </td>
+          <td>
+            <ReleaseLabelList labels={release.labels} />
+          </td>
+          <td>
+            <ReleaseCatnoList labels={release.labels} />
+          </td>
+          <td className="barcode-cell">{formatBarcode(release.barcode)}</td>
+          {$c.session && $c.session.tport
+            ? <td><TaggerIcon entity={release} /></td>
+            : null}
+        </tr>
+      ))}
+    </>
+  );
+}
+
+const ReleaseGroupIndex = ({
+  $c,
+  eligibleForCleanup,
+  pager,
+  mostPopularReview,
+  mostRecentReview,
+  numberOfRevisions,
+  releaseGroup,
+  releases,
+  wikipediaExtract,
+}: Props) => (
+  <ReleaseGroupLayout entity={releaseGroup} page="index">
+    {eligibleForCleanup ? (
+      <p className="cleanup">
+        {l(
+          `This release group has no relationships or releases associated,
+           and will be removed automatically in the next few days. If this
+           is not intended, please add more data to this release group.`,
+        )}
+      </p>
+    ) : null
+    }
+    <Annotation
+      annotation={releaseGroup.latest_annotation}
+      collapse
+      entity={releaseGroup}
+      numberOfRevisions={numberOfRevisions}
+    />
+    <WikipediaExtract
+      cachedWikipediaExtract={wikipediaExtract || null}
```

Comment:
The above type change would also make `|| null` unnecessary here:
```suggestion
      cachedWikipediaExtract={wikipediaExtract}
```